packages/mermaid/.lintstagedrc.mjs

```diff
@@ -4,4 +4,5 @@ export default {
   'src/docs/**': ['pnpm --filter mermaid run docs:build --git'],
   'src/docs.mts': ['pnpm --filter mermaid run docs:build --git'],
   'src/(defaultConfig|config|mermaidAPI).ts': ['pnpm --filter mermaid run docs:build --git'],
+  'src/schemas/config.schema.yaml': ['pnpm --filter mermaid run types:verify-config'],
```

Comment:
Yes it should :) Fixed in https://github.com/mermaid-js/mermaid/commit/c152b0af558798db165c127b4cce16417cbc68d2

I also had to add a `--git` flag too, to also do the `git add ./src/config.type.ts` bit.

In a future PR, I should extract out the common bits of `./src/docs.mts` and move it into something like `./scripts/script-helpers.mts`, since I feel like I'm re-implementing a lot of the code from `./src/docs.mts`.